Mathlib/LinearAlgebra/RootSystem/GeckConstruction/Semisimple.lean

```diff
@@ -180,6 +180,8 @@ open LieModule Matrix
 
 local notation "H" => cartanSubalgebra' b
 
+set_option maxHeartbeats 210000 in
+-- This declaration was already right at the heartbeats limit
```

Comment:
To elaborate: right now, it takes between 195000 and 196000 heartbeats. `contrapose!` becoming slightly slower pushes (no pun intended) it over the limit.